### PR TITLE
Minor changes to the README

### PR DESCRIPTION
**Pull request by mleeds95**
[It appears](https://support.google.com/voice/troubleshooter/1279544?hl=en#ts=1279547,1279551) Google Voice no longer allows you to receive messages from websites so it can't be used for adding a number to a Twitter account.

### CODE REVIEW

**Comment by ohnorobo** [comment]
There is a twitter-internal way to use the same phone number for multiple bots. It's described [here](http://blog.mollywhite.net/twitter-bots-pt2/). (Search for "I need to transfer an API key to another account".)

**Comment by mleeds95** [comment]
Thanks for the tip @ohnorobo

File: README.md [hunk 1]
```diff
@@ -10,8 +10,14 @@ Example twitter feed: [http://www.twitter.com/ericapets](http://www.twitter.com/
 
 ## Set Up for Local Development
 
+### System Requirements
```
Comment (boonrs) [inline README.md]: My preference is not to include gems in the readme and let the Gemfile serve that purpose. It eliminates the need to keep them in sync.

File: README.md [hunk 2]
```diff
@@ -37,7 +43,7 @@ Edit a the local .env file by adding the appropriate config values. How to get T
     2. `heroku config:set api_secret="your_consumer_secret_key_goes_here"`
     3. `heroku config:set access_token="your_access_token_goes_here"`
     4. `heroku config:set access_token_secret="your_access_token_secret_goes_here"`
-5. Configure pet datasource–you have two options (below). Search on Petfinder for your shelter and use that if you find it, otherwise look on PetHarbor. 
+5. Configure pet datasource–you have two options (below). Search on Petfinder for your shelter and use that if you find it, otherwise look on PetHarbor. If you want to pull data from multiple shelters (choosing at random), make the shelter id variable a comma-separated list.
```
Comment (boonrs) [inline README.md]: Does this apply to both Petfinder and PetHarbor? I'm not seeing it in the documentation for the sites.

Comment (mleeds95) [inline README.md]: That happens before the API request is sent. See [here](https://github.com/codeforamerica/CutePets/blob/master/lib/cute_pets/pet_fetcher.rb#L128).

Comment (boonrs) [inline README.md]: :foggy: Ah! Thanks for the code refresher. :ocean:

**Comment by boonrs** [comment]
Thanks for your contribution @mleeds95! :bow:

**Pull request merged**